tests/parser/tests/cases/should_pass/if_branches/case.hash

```diff
@@ -0,0 +1,7 @@
+if b == 2 {
+     print("b is 2")
+} else if b == 3 {
+     print("b is 3")
+} else {
+    print("b isn't 2 or 3 ")
+}
```

Comment:
Is this meant to be an expression? How does it pass without semis?

Reply:
well because it is treated as a block and thus it becomes an expression in the block. It might also be the case that it is at the end of the file.

Reply:
But doesn't an expression in statement position (since this is a module [right?]) require a semicolon at the end?

Reply:
Not if it's a block